.nvmrc

```diff
@@ -1 +1 @@
-11.4.0
+15.9.0
```

Comment:
make sure to look for versions in all the places: package.json>engine, gitlab runners, dockerfiles, docs dir